jovian/jovian_nb_ext/main.js

```diff
@@ -575,45 +532,115 @@ define([
-                : $($("input[name=secret_opt")[0]).prop("checked", true);
-              jvn_params.capture_env == "False"
-                ? $($("input[name=cap_opt")[1]).prop("checked", true)
-                : $($("input[name=cap_opt")[0]).prop("checked", true);
-              jvn_params.create_new == "False"
-                ? $($("input[name=create_opt")[1]).prop("checked", true)
-                : $($("input[name=create_opt")[0]).prop("checked", true);
-              jvn_params.env_type == "conda"
-                ? $("#env_opt option:contains('conda')").prop("selected", true)
-                : $("#env_opt option:contains('pip')").prop("selected", true);
+        open: async function() {
+          let project_id_helper = async () => {
+            if (getParams() == null) {
+              return;
+            }
+            let project_id = getParams().project_id;
+            let j_id = "#project_id_box";
+            if (project_id.length != 0 && !$(j_id).prop("disabled")) {
+              $(j_id).val(project_id);
+              return;
             }
+            let project_title = "";
+            await getProjectTitle().then(title => {
+              if (title != undefined) {
+                project_title = title;
+              }
+              if (!$(j_id).prop("disabled")) {
+                $(j_id).val(project_title);
+              }
+            });
+          };
+
+          let git_message_helper = () => {
+            if (!$("#git_msg_box").prop("disabled")) {
+              $("#git_msg_box").val($("#project_msg_box").val());
+            }
+          };
+
+          let params = getParams();
+          if (params != null) {
+            $("#project_msg_box").val(params.message);
+            $("#nb_filename_box").val(params.filename);
+            $("#files_box").val(params.files);
+            $("#env_opt option:contains(" + params.environment + ")").prop(
+              "selected",
+              true
+            );
+            await project_id_helper();
+            $("#if_new option:contains(" + params.new_project + ")").prop(
+              "selected",
+              true
+            );
+            $("#nb_opt option:contains(" + params.privacy + ")").prop(
+              "selected",
+              true
+            );
+            $("#artifacts_box").val(params.outputs);
+            $("#if_git option:contains(" + params.git_commit + ")").prop(
+              "selected",
+              true
+            );
+            // $("#git_msg_box").val(params.git_message);
+            git_message_helper();
+          } else {
+            $("#nb_filename_box").val(
+              Jupyter.notebook.notebook_name.replace(".ipynb", "")
+            );
+            $("#env_opt option:contains('auto')").prop("selected", true);
+            $("#if_new option:contains('False')").prop("selected", true);
+            $("#if_git option:contains('False')").prop("selected", true);
```

Comment:
Set this to True, as `git_commit=True` by default in the library. This will create ambiguity.

Also remove the disabled git_message box, similar to previous comment this is a edge case.

Improvement can be reading whether its inside a git repo, but we can discuss after the launch.